osu.Game/Rulesets/Objects/ConvertHitObject.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Rulesets.Objects
+{
+    /// <summary>
+    /// A hit object only used for conversion, not actual gameplay.
+    /// </summary>
+    public class ConvertHitObject : HitObject
```

Comment:
As above, not really a fan of this. What's the reasoning here?

Reply:
As discussed before:
1. Make `abstract`.
2. Move to legacy namespace.
3. `internal`ise.
4. Make `HitObject` non-abstract again, give it sane default implementations.